Tests/App/config.yml

```diff
@@ -0,0 +1,76 @@
+framework:
+    secret:          secret
+    test: ~
+    router:          { resource: "%kernel.root_dir%/../App/routing.yml" }
+    session:
+        handler_id:  ~
+        storage_id: session.storage.mock_file
+        name: MOCKSESSID
+
+monolog:
+    handlers:
+        main:
+            type:  stream
+            path:  "%kernel.logs_dir%/%kernel.environment%.log"
+            level: debug
+
+security:
+    providers:
+        HWI\Bundle\OAuthBundle\Tests\App\UserProvider:
+            id: HWI\Bundle\OAuthBundle\Tests\App\UserProvider
+    firewalls:
+        login_area:
+            pattern: ^/(login|connect)
```

Comment:
Now the test is working, in the sense that is failing due to #1522 as expected. But this line concerns me: I was forced to put `/connect` under the anonymous part of the firewall, but I didn't do that in my personal project. Is it correct?

Reply:
mm I guess that it's needed because the is no firewall that admit anonymous users and matches the `/connect` pattern, so it will keep redirecting to `/login`.